From 6507fbb4e41b4d49b5154e585672aeec0d12b950 Mon Sep 17 00:00:00 2001
From: Hai Nguyen Pham <hai.pham.ud@renesas.com>
Date: Thu, 11 Oct 2018 18:02:14 +0700
Subject: [PATCH 694/909] Revert "arm64: dts: renesas: r8a7795-es1: Enable
 IPMMU-MP1"

commit 3bf0cbcdf2ca8ae5276a8727f74fae726893e916 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This reverts commit 26c5f2a4f3c4a17f2a08112de7bf35b9a73b652b
to keep all IPMMU devices are disabled by default,
maintain the same IPMMU support environment with previous
released R-Car Linux BSP.

Signed-off-by: Hai Nguyen Pham <hai.pham.ud@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi b/arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi
index 508a7ac1cef2..a4b8a50c0f3a 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi
@@ -29,6 +29,7 @@
 		reg = <0 0xec680000 0 0x1000>;
 		renesas,ipmmu-main = <&ipmmu_mm 5>;
 		#iommu-cells = <1>;
+		status = "disabled";
 	};
 
 	ipmmu_sy: mmu@e7730000 {
-- 
2.17.1

